lesson06/assignment/css/_components.content.scss

```diff
@@ -8,6 +8,292 @@ html {
     box-sizing: inherit;
 }
 
+.index {
+  grid-template-areas:
+    "header"
+    "nav"
+    "mainLanding"
+    "footer"
+}
+
+
+
+.header {
+  grid-area: header;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
+
+  &__lockup {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid green;
+    &__logo {
+      position: relative;
+      left: 0%;
+      color: white;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background-color: #00c3a899;
+      border-radius: 50%;
+      width: 7rem;
+      height: 7rem;
+      margin-right: 1rem;
+    }
+    &__title {
+      margin: auto;
+    }
+  }
+
+}
+
+@media only screen and (max-width: 510px) {
+  .header__logo, .footer__logo {
+    width: 4rem;
+    height: 4rem;
+  }
+}
+@media only screen and (max-width: 460px) {
+  .header, .footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    &__lockup {
+      flex-direction: column;
+      &__logo {
+        width: 7rem;
+        height: 7rem;
+      }
+    }
+
+  }
+}
+// ~~~~~NAV~~~~~
+nav {
+  width: auto;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  border: 2px solid blue;
```

Comment:
I think you have a lot of border styles that were useful debugging tools while coding this content, but don't add to the design. Those should be deleted before final code submission.